### ISSUE TITLE
Getting kicked by root (Chatroom closed by server) in a groupchat

### ISSUE THREAD

**Issue by linkuso** [bitlbee/bitlbee-facebook#183]
Hi,
I have a strange issue where it seems randomly I'm getting kicked from a groupchat, like yesterday one message was written 20:46 nothing after that until 09:23 the morning after and around 21:08 i get root has kicked <user> (Chatroom closed by server).

This happens pretty frequently, it happend the same day as 20:46 but ish 20h before that at 00:18, same error root has kicked <user> (Chatroom closed by server).

How can i provide more details on what actually happened? Thanks!

**Comment by dequis** [comment]
Read &bitlbee at that time for relevant error messages.

**Comment by linkuso** [comment]
Finally go a error message.

08:55:53       @root | Could not take over old session

**Comment by dequis** [comment]
That means your irc client disconnected from bitlbee, likely due to internet connection issues between both sides (if it's local, it's a bug).

See `help set allow_takeover`, if you disable that feature it won't try to take over and just reconnect cleanly.

Still unlikely to be related to the original issue you had, though, since getting kicked by bitlbee happens on IM account disconnection, not on irc disconnection

**Comment by linkuso** [comment]
This is local, bitlbee and my client (Weechat) is on the same machine. 

Will try to disable anyhow and see if it makes any difference.

**Comment by dequis** [comment]
Yeah that's tricky then, you're probably missing other more relevant errors before that one.

**Comment by linkuso** [comment]
Can i somehow enable more logging or are there more log's saved elsewhere?

**Comment by dequis** [comment]
It's stuff that should be already visible in your client's buffers, either the server one or &bitlbee. It disconnected from bitlbee, and there should be messages before that disconnection.

**Comment by linkuso** [comment]
got another "disconnect" this time it was 

**2019-04-06 17:52:49     @root   facebook - Error: Failed to read fixed header
2019-04-06 17:52:49     @root   facebook - Signing off..**

Can i somehow make it so that when it got connection to facebook again it will join the groupchats automatically? That would (however a ugly way to fix it) work.

**Comment by dequis** [comment]
`channel #something set auto_join true`
